### ISSUE TITLE
[FEATURE REQUEST] Create a desktop version of this app

### ISSUE THREAD

**Issue by gavinkalika** [Protoqol/Prequel#17]
**Is your feature request related to a problem? Please describe.**
It would be nice to have a desktop version of this application similar to sequel pro

**Comment by QuintenJustus** [comment]
Hey, your feature requests are pretty good. Why’d you close them?

**Comment by gavinkalika** [comment]
reopened 😄

**Comment by gavinkalika** [comment]
what are your thoughts on implementation?

An electron app?

**Comment by QuintenJustus** [comment]
Maybe, electron is still a bit heavy on many systems. First a Prequel standalone will be developed, and with that also a standalone desktop app. So still a long way to go, but certainly worth looking into with enough support! Thanks for your request!

**Comment by xuanquynh** [comment]
The native application is nice but a standalone and stable web-based application has to be done first. :)

**Comment by QuintenJustus** [comment]
Closing now, has been added to roadmap. Thanks for your fr!
